pKa_challenge_instructions.md

```diff
@@ -158,6 +167,9 @@ These and blank lines will be ignored.
 
 The file must contain the following four components in the following order: your predictions, a name for your computational protocol, a list of the major software packages used, and a long-form methods description. 
 Each of these components must begin with a line containing only the corresponding keyword: `Predictions:`, `Name:`, `Software:`, and `Method:`, as illustrated in the example files. 
+
+Example submission files can be found in `physical_properties/pKa/example_submission_files/` directory to illustrate expected format when filling submission templates of the pKa challenge.
```

Comment:
If this directory existed before this PR, could you make this a hyperlink so people can click on the path?

Reply:
I added a link.